Source/santametricservice/SNTMetricServiceTest.m

```diff
@@ -16,6 +17,9 @@ @interface SNTMetricServiceTest : XCTestCase
 @property id mockConfigurator;
 @property NSString *tempDir;
 @property NSURL *jsonURL;
+@property id mockSession;
```

Comment:
These were moved to be properties to avoid a bug an issue with OCMock and releasing the blocks too many times.